recipes/ecfeed/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.18)
+
+project(example CXX)
+
+set(CMAKE_CXX_COMPILER_VERSION 8)
+
+add_compile_options(-std=c++2a)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(example example.cpp)
+
+target_link_libraries(${PROJECT_NAME} stdc++fs)
+target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
```

```suggestion
include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
conan_basic_setup(TARGETS)

find_package(ecfeed CONFIG REQUIRED)

add_executable(example example.cpp)
target_link_libraries(${PROJECT_NAME} ecfeed::ecfeed)
set_target_properties(${PROJECT_NAME} PROPERTIES CXX_STANDARD 20)
```

Use the target provided by Conan, it will be default for Conan 2.0
Do not link system libraries here. If they are required, the package should define it directly as [system_libs](https://docs.conan.io/en/latest/reference/conanfile/attributes.html#cpp-info)

---

Updated.

---

I was able to build and test the project locally, however something went wrong while building it on the server. I'll fix this at the beginning of the next week. It has something to do with system libraries. Thanks for the explanation and the reference!

---

It seems be a limitation in the docker image `conanio/clang11`. I'll check a hotfix

---

Checking again, filesystem is under experimental, so it would need to include <experimental/filesystem>. It occurs due the c++ library version installed